apps/api/src/app/domains/images/images.controller.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
remove this comment

Reply:
third part api using camelcase, so this comment on this case is required